Neos.Neos/Classes/ViewHelpers/Backend/JavascriptConfigurationViewHelper.php

```diff
@@ -105,28 +105,35 @@ public function injectSettings(array $settings)
     public function render()
     {
         $configuration = [
```

Comment:
Is it not possible to defined the configuration as array and do a json_encode at the end?

Reply:
As we here just write JS code in php and that is not just JSON I guess it is not that easy.
I did not adjusted that much here.